js/index.ts

```diff
@@ -125,22 +125,11 @@ const boxes = {
 };
 
 /*
-if you think that you can do this in some other way, you are likely wrong.
-
-Myst tried to do it 2 times, and both failed. The issues were difficult to
-reproduce. For example, last time I wrote something to the editor, then opened
-the draw area. I pressed ctrl+z, and it undid in the draw area as expected.
-Then I pressed ctrl+z again and it undid in the editor instead of the draw
-area, even though the draw area was open and I hadn't touched the editor in any
-way (wtf). The bug reproduced many times again, so to undo many times in the
-draw area, I had to open the draw area, ctrl+z, close draw are, open draw area,
-ctrl+z, close draw area etc many times. A little while later I tried the same
-thing again and I couldn't reproduce the problem (wut).
-
-PLEASE DON'T REWRITE THIS VERY DIFFERENTLY unless you truly understand what is
-going on, unlike anyone else who has worked on mathpaste so far!
-*/
```

Comment:
Keep this old comment. It's really worth emphasizing that the logic should not be completely changed.

Reply:
Actually I don't think we should keep a long and ugly legacy comment around. However, make it clear in your comment that this isn't as easy as it looks like, and most of the time shouldn't be touched.